upload.go

```diff
@@ -34,15 +34,22 @@ func Upload(file *os.File) (string, error) {
 	}
 
 	cfg := aws.NewConfig().WithRegion(awsRegion).WithCredentials(creds)
-	svc := s3.New(session.New(), cfg)
+	sesh, err := session.NewSession()
```

Comment:
session.New is deprecated https://docs.aws.amazon.com/sdk-for-go/api/aws/session/